downstream/modules/aap-hardening/con-platform-components.adoc

```diff
@@ -3,7 +3,7 @@
 
 [id="con-platform-components_{context}"]
 
-= {PlatformName} components
+= {PlatformNameShort} components
 
```

Comment:
Noticed link in line 14 of this file points to the wrong id and should be updated to:
link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_planning_guide/ref-aap-components